events/2025-01-28-src.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: event
+
+# Status of this page, remove these lines when ready
+draft: true  # will hide your event from the GTN events list
+status: wip  # add a banner that warns that the contents of the page are still subject to change
+
+# Description of your event
+title: Galaxy at SURF Research Cloud workshop
+description: |
+  Half-day workshop demonstrating the local Galaxy instance built for users of SURF Research Cloud. 
+
+#cover-image:         # image for your corse, put in 'events/images' folder
```

Comment:
would you like to use the SURF logo image you included for the tutorial on the event page, or happy with the default GTN image?